docs/docs-content/automation/palette-cli/commands/validate-auth.md

```diff
@@ -0,0 +1,351 @@
+--------
+- Condition StringLike: iam:AWSServiceName=[autoscaling.amazonaws.com];  not applied to action(s) [iam:CreateServiceLinkedRole] for resource arn:*:iam::*:role/aws-service-role/autoscaling.amazonaws.com/AWSServiceRoleForAutoScaling from policy PaletteControllerPolicy
+- v1alpha1.IamUserRule PaletteClusterOperator missing action(s): [autoscaling:CreateAutoScalingGroup autoscaling:DeleteAutoScalingGroup] for resource arn:*:autoscaling:*:*:autoScalingGroup:*:autoScalingGroupName/* from policy PaletteControllerPolicy
+- v1alpha1.IamUserRule PaletteClusterOperator missing action(s): [s3:DeleteObject] for resource arn:*:s3:::* from policy PaletteControllerPolicy
+- v1alpha1.IamUserRule PaletteClusterOperator missing action(s): [secretsmanager:CreateSecret secretsmanager:DeleteSecret] for resource arn:*:secretsmanager:*:*:secret:aws.cluster.x-k8s.io/* from policy PaletteNodesPolicy
+```
+
+Use the output to help you address the validation failures. In this example, the Validator identified two types of IAM
+permission issues, which are missing actions and missing conditions.
+
+To resolve the missing actions, add the missing IAM permissions to the **PaletteClusterOperator** IAM user. The
+following table summarizes the missing actions and their required resource scopes, identifying which Palette policy
+should contain these permissions.
+
+| **IAM Policy**              | **Missing Actions**                                                              | **Resource Scope**                                                |
+| --------------------------- | -------------------------------------------------------------------------------- | ----------------------------------------------------------------- |
+| **PaletteControllerPolicy** | `autoscaling:CreateAutoScalingGroup` <br /> `autoscaling:DeleteAutoScalingGroup` | `arn:*:autoscaling:*:*:autoScalingGroup:*:autoScalingGroupName/*` |
+| **PaletteControllerPolicy** | `s3:DeleteObject`                                                                | `arn:*:s3:::*`                                                    |
+| **PaletteNodesPolicy**      | `secretsmanager:CreateSecret` <br /> `secretsmanager:DeleteSecret`               | `arn:*:secretsmanager:*:*:secret:aws.cluster.x-k8s.io/*`          |
+
+To resolve the missing conditions, apply the required conditions to the specified actions. From interpreting the example
+extract below, the Validator detected that the **PaletteControllerPolicy** IAM policy requires a condition to be added
+for the `iam:CreateServiceLinkedRole` action to restrict service-linked role creation specifically to Auto Scaling
+services.
+
+```yaml hideClipboard
+- Condition StringLike:
+    iam:AWSServiceName=[autoscaling.amazonaws.com];  not applied to action(s) [iam:CreateServiceLinkedRole] for resource
+    arn:*:iam::*:role/aws-service-role/autoscaling.amazonaws.com/AWSServiceRoleForAutoScaling from policy
+    PaletteControllerPolicy
+```
+
+##### Azure
+
+<!-- TBC -->
```

Comment:
@benradstone What is this for? This is not something we use.

Reply:
Forgot to take this out, it was for my own awareness to update once I got a full results output.

Reply:
```suggestion
```